code/controller/controller.h

```diff
@@ -15,10 +15,10 @@ Q_OBJECT
     };
 
     enum Dir {
-        RIGHT, // +X

```

Comment:
I need to check if the order of the directions here has any importance

Reply:
I don't think it does, the only other uses of this enum got the values by name rather than index, so changing the order shouldn't affect anything. I didn't notice anything behaving funny when testing either.